helper/schema/resource_data_test.go

```diff
@@ -3915,6 +3919,162 @@ func TestResourceData_nonStringValuesInMap(t *testing.T) {
 	}
 }
 
+func TestResourceDataGetRawConfigAt(t *testing.T) {
```

Comment:
Another test to consider here is when the attribute is valid in the schema, but the value is actually null (should be  `cty.NullVal(<cty.Type>)` and not be `cty.DynamicVal`)